### PR TITLE
Added getblockchainsyncstatus check and Node Just started

### PR DESCRIPTION
**Pull request by ciripel**

### CODE REVIEW

File: src/__xsg/info.lua [hunk 1]
```diff
@@ -80,11 +80,26 @@ if _info.snowgemd == "running" then
...
         function () -- synchronization check
-            local _exitcode = _exec_xsg_cli("getblocktemplate")
-            _info.synced = _exitcode == 0
-            if not _info.synced then
+            local _exitcode, _stdout, stderr = _exec_xsg_cli("getblockchainsyncstatus")
+            local _success, _output = _get_xsg_cli_result(_exitcode, _stdout, _stderr)
+
+            if _success then
+                if _output.IsBlockchainSync = true
+                    _info.level = "ok"
```
Comment (cryi) [inline src/__xsg/info.lua]: `_info.level` has to be set to `ok` only once and that is before any of checks happen. (It `ok` by default.) So not here 🙂

**Review by cryi — CHANGES_REQUESTED** [review]
Just remove mentioned assignment please. 

*This can be merged only after new daemon is released.*

File: src/__xsg/info.lua (hunk 1 above)
Comment (cryi) [reply src/__xsg/info.lua]: Please remove others `_info.level = "ok"` after this one as well.

**Comment by ciripel** [comment]
Resolved

**Comment by cryi** [comment]
Thank you 🙂

**Pull request merged**